tests/src/test/scala/common/rest/WskRestOperations.scala

```diff
@@ -733,13 +738,23 @@ class RestActivationOperations(implicit val actorSystem: ActorSystem)
                    fieldFilter: Option[String] = None,
                    last: Option[Boolean] = None,
                    summary: Option[Boolean] = None)(implicit wp: WskProps): RestResult = {
-    val rr = activationId match {
+    val actId = activationId match {
+      case Some(id) => activationId
+      case None =>
+        last match {
```

Comment:
Caution here is that last night be stale.

Reply:
Sorry?

Reply:
The value of the “last” activation might be the last id indexed not necessarily the last id stored. This is fine for your test related changes from what I scanned but felt it necessary to mention it as this behavior of last isn’t generally obvious.